_docs/configuration_files/s3.rest

```diff
@@ -152,6 +152,78 @@ For more details refer to: http://boto3.readthedocs.io/en/latest/reference/servi
               }
           ]
 
+``lifecycle``
+*************
+
+S3 Lifecycle configuration block
+
+``lifecycle`` *Keys*
+^^^^^^^^^^^^^^^^^^^^
+
+``enabled`` : Enables/Disables S3 Lifecycle configuration
+
+    | *Type*: boolean
+    | *Default*: ``false``
+
+``lifecycle_rules`` :
+
+A list of S3 lifecycle rules, if a lifecycle exists it replaces it.
+For more details refer to: http://boto3.readthedocs.io/en/latest/reference/services/s3.html#S3.Client.put_bucket_lifecycle_configuration
+
+    | *Type*: object
+    | *Default*: ``None``
+    | *Example config*:
+
+      .. code-block:: json
+
+          [
+            {
+                'Expiration': {
+                    'Date': datetime(2015, 1, 1),
+                    'Days': 123,
+                    'ExpiredObjectDeleteMarker': True|False
+                },
+                'ID': 'string',
+                'Prefix': 'string',
+                'Filter': {
+                    'Prefix': 'string',
+                    'Tag': {
+                        'Key': 'string',
+                        'Value': 'string'
+                    },
+                    'And': {
+                        'Prefix': 'string',
+                        'Tags': [
+                            {
+                                'Key': 'string',
+                                'Value': 'string'
+                            },
+                        ]
+                    }
+                },
+                'Status': 'Enabled'|'Disabled',
+                'Transitions': [
+                    {
+                        'Date': datetime(2015, 1, 1),
+                        'Days': 123,
+                        'StorageClass': 'GLACIER'|'STANDARD_IA'|'ONEZONE_IA'
+                    },
+                ],
+                'NoncurrentVersionTransitions': [
+                    {
+                        'NoncurrentDays': 123,
+                        'StorageClass': 'GLACIER'|'STANDARD_IA'|'ONEZONE_IA'
+                    },
+                ],
+                'NoncurrentVersionExpiration': {
+                    'NoncurrentDays': 123
+                },
+                'AbortIncompleteMultipartUpload': {
+                    'DaysAfterInitiation': 123
+                }
+            }
+          ]
```

Comment:
I see above is aws docs and simple to understand but do we want to provide an example policy as well? One simple example?

Reply:
I think this example is missing the top key maybe?

```
{
  "lifecycle_rules": [
      { "Expiration": ... }
    ],
    [ 
      { "Expiration": ... }
    ]
}
```

Reply:
@sijis, the top key is missing on purpose, which is misleading. It's handled in the Function, https://github.com/gogoair/foremast/pull/289/files?diff=unified#diff-29843ed9507806c7194f2bf847faccd3R197.

And I agree with @bazimov, I need a simple, sane copy and paste example in JSON (the current example is in broken Python).

Reply:
I either opt for leaving it as is, or removing it completely. There are far too many fields to provide an accurate example. Not including all the potential options of a single rule can be further misleading to users. Ultimately, this should be sources from the docs so I am leaning more towards removing it.